frontend/src/features/permits/components/dashboard/ApplicationStepPage.tsx

```diff
@@ -72,14 +72,13 @@ export const ApplicationStepPage = ({
     applicationData?.permitType,
   );
 
-  // Permit must be an application in order to allow application-related steps
-  // (ie. empty status for new application, or in progress or incomplete payment status)
+  // Permit must be an application in progress in order to allow application-related edit/review/add to cart steps
+  // (ie. empty status for new application, or in progress)
   const isValidApplicationStatus = () => {
     return (
       !isInvalidApplication &&
       (!applicationData?.permitStatus ||
-        applicationData?.permitStatus === PERMIT_STATUSES.IN_PROGRESS ||
-        applicationData?.permitStatus === PERMIT_STATUSES.WAITING_PAYMENT)
+        applicationData?.permitStatus === PERMIT_STATUSES.IN_PROGRESS)
```

Comment:
For this particular situation, we do need to include ```!applicationData?.permitStatus``` before the || since we need to consider creating new applications (which have no permit status yet upon initial load). We are allowing all applications with IN_PROGRESS (application already created/edited before) as well as no status (new applications with initial empty fields) to proceed to form/review pages.

Reply:
Ah I see. Thanks!